tutorials/xr/xr_full_screen_effects.rst

```diff
@@ -0,0 +1,81 @@
+.. _doc_xr_full_screen_effects:
+
+XR full screen effects
+======================
+
+When adding custom full screen effects to your XR application, one valid approach is
+applying a full screen quad to the XR camera and applying effects to that quad's shader.
+
+.. image:: img/xr_full_screen_effects_starting_quad.webp
+
+Here is a simple vignette shader that you might apply to the quad, darkening the edges
+of a user's screen:
+
+.. code-block:: glsl
+
+  shader_type spatial;
+  render_mode depth_test_disabled, skip_vertex_transform, unshaded, cull_disabled;
+
+  void vertex() {
+	  POSITION = vec4(VERTEX.xy, 1.0, 1.0);
+  }
+
+  void fragment() {
+	  ALBEDO = vec3(0.0);
+	  ALPHA = dot(UV * 2.0 - 1.0, UV * 2.0 - 1.0) * 2.0;
+  }
```

Comment:
I'm not sure about including the incorrect shader code in full, because folks may not read very closely and just copy-paste this.

I think it may be better to only show full shader code for the correct one, and maybe put comments calling out the part that adjusts for the asymmetric projection matrix? Saying like, "if you remove this, the effect won't be correctly centered")